ng2-components/ng2-alfresco-core/src/components/collapsable/accordion-group.component.spec.ts

```diff
@@ -75,25 +75,26 @@ describe('AccordionGroupComponent', () => {
         });
     });
 
-    it('should hide expand icon by default', () => {
+    it('should show expand icon by default', () => {
```

Comment:
In the previous approach (https://github.com/Alfresco/alfresco-ng2-components/commit/22bbffdb2162cb3ed670c5797dad7a54e0ac3d27), accordion icon hiding logic was solely based on the length of the inner HTML and when we used this with ADF Dashboard, we faced issues when the child elements were conditionally added with ng-if directive. So we thought of having an input and let the developer decide when the expand icon needs to be hidden.

The new input (hasAccordionIcon) is optional and the default value is true, so default behaviour is showing the expand icon and the same is hidden only when the hasAccordionIcon input is explicitly set as false.